docs/advanced-entity-analytics/entity-risk-scoring.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[entity-risk-scoring]]
+= Entity Risk Scoring
+
+beta::[]
+
+Entity Risk Scoring is an advanced {elastic-sec} analytics feature that helps security analysts detect changes in an entity's risk posture, hunt for new threats, and prioritize incident response.
+
+Entity Risk Scoring allows you to monitor the change in the risk posture of hosts and users from your environment. The risk scoring engine generates these advanced scoring analytics by factoring threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.
```

Comment:
```suggestion
Entity Risk Scoring allows you to monitor changes in the risk score of hosts and users in your environment. The risk scoring engine assesses risk based on data from various sources.
```
The sentence with "factoring" wasn't quite holding together. I'm not sure what all sources it uses so I just suggested various.

Reply:
How does this sound @benironside, @SourinPaul?  
```suggestion
Entity Risk Scoring allows you to monitor changes in the risk score of hosts and users in your environment. When generating advanced scoring analytics, the risk scoring engine takes into account threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.
```

Reply:
@natasha-moore-elastic this is good, just one tweak. 
```suggestion
Entity Risk Scoring allows you to monitor risk score changes of hosts and users in your environment. When generating advanced scoring analytics, the risk scoring engine utilizes threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.
```

Reply:
Reads great!